src/analysis/function_parameters.rs

```diff
@@ -240,7 +240,12 @@ pub fn analyze(
             }
         };
 
-        if async_func && to_remove.contains(&(pos - correction_instance)) {
+        if async_func
+            && pos
+                .checked_sub(correction_instance)
+                .map(|p| to_remove.contains(&p))
+                .unwrap_or(false)
+        {
```

Comment:
Are these `checked_sub` related to https://github.com/gtk-rs/gir/pull/1427/files ?

Reply:
Seems to be a similar issue, but I was getting panics just running against gtk-rs-core

Reply:
Resolving since that one is merged now